wire/protocol.go

```diff
@@ -93,18 +93,23 @@ const (
 	// SFNode2X is a flag used to indicate a peer is running the Segwit2X
 	// software.
 	SFNode2X
+
+	// SFNodeNetWorkLimited is a flag used to indicate a peer supports serving
```

Comment:
Ah, if this is an actual p2p requirement that we need to be able to serve at least the last 288 blocks, I think we should update our minimum prune value accordingly (see previous comment of not using the max block file size as the minimum).

Reply:
Refer to the reply here: https://github.com/btcsuite/btcd/pull/1971#discussion_r1200443717

Reply:
I believe `SFNodeNetworkLimited` is untyped here